AVOutput/AVOutputTV.cpp

```diff
@@ -0,0 +1,5263 @@
+            LOGINFO("Exit\n");
+            returnResponse(true);
+        }
+    }
+
+    uint32_t AVOutputTV::getBrightness(const JsonObject& parameters, JsonObject& response)
+    {
+        LOGINFO("Entry");
+
+        std::string pqmode;
+        std::string source;
+        std::string format;
+        std::string key;
+        int sourceIndex=0,pqIndex=0,formatIndex=0;
+        int brightness = 0;
+
+        if (parsingGetInputArgument(parameters, "Brightness",source, pqmode, format) != 0)
+        {
+            LOGINFO("%s: Failed to parse argument\n", __FUNCTION__);
+            returnResponse(false);
+        }
+
+        if (getParamIndex(source,pqmode,format,sourceIndex,pqIndex,formatIndex) == -1) 
+	{
+            LOGERR("%s: getParamIndex failed to get \n", __FUNCTION__);
+            returnResponse(false);
+        }
+
+        int err = getLocalparam("Brightness",formatIndex,pqIndex,sourceIndex,brightness, PQ_PARAM_BRIGHTNESS);
+        if( err == 0 ) 
+	{
+            response["brightness"] = brightness;
+            LOGINFO("Exit : Brightness Value: %d \n", brightness);
+            returnResponse(true);
+        }
+        else 
+	{
+            returnResponse(false);
+        }
+    }
+
+    uint32_t AVOutputTV::setBrightness(const JsonObject& parameters, JsonObject& response)
+    {
+        LOGINFO("Entry\n");
+
+        std::string value;
+        std::string pqmode;
+        std::string source;
+        std::string format;
+        int brightness = 0;
+	tvError_t ret = tvERROR_NONE;
+
+        value = parameters.HasLabel("brightness") ? parameters["brightness"].String() : "";
+        returnIfParamNotFound(parameters,"brightness");
+        brightness = stoi(value);
+
+        if (validateIntegerInputParameter("Brightness",brightness) != 0)
+        {
+            LOGERR("Failed in Brightness range validation:%s", __FUNCTION__);
+            returnResponse(false);
+        }
+
+        if (parsingSetInputArgument(parameters, "Brightness",source, pqmode, format) != 0)
+        {
+            LOGERR("%s: Failed to parse the input arguments \n", __FUNCTION__);
+            returnResponse(false);
+        }
+
+        if( !isCapablityCheckPassed( pqmode, source, format, "Brightness" )) 
+	{
+            LOGERR("%s: CapablityCheck failed for Brightness\n", __FUNCTION__);
+            returnResponse(false);
+        }
+
+        if( isSetRequired(pqmode,source,format) ) 
+	{
+             LOGINFO("Proceed with %s \n",__FUNCTION__);
+             ret = SetBrightness(brightness);
+        }
+
+	if(ret != tvERROR_NONE) 
+	{
+            LOGERR("Failed to set Brightness\n");
+            returnResponse(false);
+        }
+        else 
+	{
+            int params[3]={0};
+            params[0]=brightness;
+            int retval= UpdateAVoutputTVParam("set","Brightness",pqmode,source,format,PQ_PARAM_BRIGHTNESS,params);
+            if(retval != 0 ) 
+	    {
+                LOGWARN("Failed to Save Brightness to ssm_data\n");
```

Comment:
retval in failure case also u returing true at line 1160

Reply:
done and addressed similar comment in other places